src/ripple/server/impl/Port.cpp

```diff
@@ -186,6 +187,25 @@ parse_Port (ParsedPort& port, Section const& section, std::ostream& log)
         }
     }
 
+    {
+        auto const lim = get (section, "limit", "unlimited");
+
+        if (!beast::ci_equal (lim, "unlimited"))
+        {
+            try
+            {
+                port.limit = boost::lexical_cast<int>(lim);
+            }
+            catch (boost::bad_lexical_cast& ex)
+            {
+                log <<
+                    "Invalid value '" << lim << "' for key " <<
+                    "'limit' in [" << section.name() << "]\n";
+                Throw<std::exception> ();
+            }
+        }
+    }
+
     populate (section, "admin", log, port.admin_ip, true, {});
```

Comment:
This code allows negative values to be specified. We might also consider limiting the value. 
Alternate suggestion:
        auto const result = section.find("limit");
        if (result.second)
        {
            auto const ul = std::stoul(result.first);
            if (ul > std::numeric_limits<std::uint16_t>::max())
            {
                log <<
                    "Invalid value '" << result.first << "' for key " <<
                    "'limit' in [" << section.name() << "]\n";
                Throw<std::exception> ();
            }
            port.limit = ul;
        }